src/views/Profile.vue

```diff
@@ -1,13 +1,218 @@
 <template>
-  <div></div>
+  <div>
+    <div class="flex justify-between">
+      <h1 class="text-6xl">{{ $t('profile.title') }}</h1>
+    </div>
+
+    <div class="px-4 py-12 sm:py-16 my-10 bg-white shadow rounded-lg sm:px-10">
+      <div class="max-w-5xl mx-auto h-auto">
+        <div class="grid grid-cols-1 sm:grid-cols-4 gap-y-8">
+          <div class="relative w-full col-span-1 sm:col-span-3 font-light">
+            <span class="block mb-1">
+              <span class="font-medium">Email</span>: {{ user.email }}
+            </span>
+            <span class="block mb-1">
+              <span class="font-medium">Created at</span>: {{ getCreatedAt }}
+            </span>
+            <span class="block">
+              <span class="font-medium">Last updated</span>: {{ getUpdatedAt }}
+            </span>
+
+            <div v-if="edit" class="relative flex sm:absolute -bottom-4 text-3xl mt-0 sm:mt-5 w-full sm:w-5/6">
+              <base-input class="w-full sm:w-1/2 mr-1" v-model="profile.first_name" :label="$t('register.name.firstName')" />
+              <base-input class="w-full sm:w-1/2" v-model="profile.last_name" :label="$t('register.name.lastName')" />
+            </div>
+            <div v-else class="relative sm:absolute bottom-0 text-3xl mt-5">
+              <span>{{ user.first_name }} {{ user.last_name }}</span>
+            </div>
+          </div>
+
+          <div class="flex flex-col">
+            <img class="rounded-full h-32 w-32 object-cover mx-auto" :src="getAvatar" :alt="$t('profile.avatar.alt')">
+            <span v-if="!edit" class="flex justify-center mt-2 sm:mt-0">
+              <base-button outline size="sm" @click="edit = !edit">
```

Comment:
when there are more than 2 props, you should format them on new lines, for eassier reading